main/__tests__/badge.test.tsx

```diff
@@ -1,30 +1,46 @@
 // Library Import
 import React, { ReactElement } from 'react';
-import {
-  render
-} from '@testing-library/react-native';
+import { RenderResult, render } from '@testing-library/react-native';
 import renderer from 'react-test-renderer';
 // Test 대상 import
-import Badge from '../components/pureComponent/badge';
+import { Badge } from '../Badge';
 
-type badgeProps ={
-  count? : number,
-  color? : string,
+interface BadgeProps {
```

Comment:
I recommend to use `BadgeProps` from the `../Badge/index.tsx` than redefine here. 
You may exporting the interface.